model/Core/Classes/SemVer.md

```diff
@@ -4,11 +4,12 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-A String constrained to the SemVer 2.0.0 specification.
+A string constrained to the SemVer 2.0.0 specification.
 
 ## Description
 
-The semantic version is a String constrained to the SemVer 2.0.0 specification.
+The semantic version is a string
+that is following the specification of Semntic Versioning 2.0.0.
```

Comment:
Please insert link to Semantic Versioniong 2.0.0 reference.

Reply:
probably

```suggestion
that is following the specification of [Semntic Versioning 2.0.0](https://semver.org/).
```